src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,25 @@
+import "./Introduce.css";
+
+const handleInput = () => {
+  console.log("input 태그 동작");
+};
+
+const handleButton = () => {
+  console.log("button 태그 동작");
+};
+
+const Introduce = () => {
+  return (
+    <div className="introduce">
+      <img src="images/avatar.jpg" alt="" />
+      <p className="name">백지율</p>
+      <p className="th">46기</p>
+      <div className="textInput">
+        <input type={"text"} onClick={handleInput} placeholder="입력"></input>
```

Comment:
```suggestion
        <input type="text" onClick={handleInput} placeholder="입력" />
```
속성에 부여하는 값이 문자열일때는 중괄호를 사용하지 않으셔도 됩니다!
그리고 Input 태그 사이에는 어떠한 값이 없기 때문에 self closing을 적용해주세요~